src/components/SettingsDialog/Components/ThemeSettingsDropdown.tsx

```diff
@@ -7,18 +7,38 @@ import {ReactComponent as DarkIcon} from "assets/icon-darkmode.svg";
 
 type ThemeSettingsDropdownProps = {
   showDropdown: boolean;
+  setShowDropdown: Dispatch<SetStateAction<boolean>>;
 };
 
-export const ThemeSettingsDropdown: VFC<ThemeSettingsDropdownProps> = ({showDropdown}) => {
-  const [theme, setTheme] = useState(document.documentElement.getAttribute("theme") ?? "light");
+enum Theme {
+  LIGHT = "light",
+  DARK = "dark",
+}
+
+export const ThemeSettingsDropdown: VFC<ThemeSettingsDropdownProps> = ({showDropdown, setShowDropdown}) => {
+  const [theme, setTheme] = useState<Theme>((document.documentElement.getAttribute("theme") as Theme) ?? Theme.LIGHT);
   useEffect(() => {
     document.documentElement.setAttribute("theme", theme!);
     localStorage.setItem("theme", theme!);
   }, [theme]);
 
+  const handleKeyDown = (newTheme: Theme) => (e: KeyboardEvent<HTMLLIElement>) => {
+    switch (e.key) {
+      case " ":
+      case "SpaceBar":
```

Comment:
See previous comment to this.
```suggestion
      case "Spacebar":
```